### PR TITLE
Role in workflow and group

### PR DESCRIPTION
**Pull request by veolys-nanne**

### CODE REVIEW

File: WorkflowFunctionAdminBundle/Form/Type/WorkflowFunctionType.php [hunk 1]
```diff
@@ -52,6 +56,10 @@ public function configureOptions(OptionsResolver $resolver)
         ));
     }
 
+    protected function getChoices()
+    {
+        return $this->roleRepositoryInterface->findWorkflowRole();
+    }
     /**
```
Comment (itkg-alavieille) [inline WorkflowFunctionAdminBundle/Form/Type/WorkflowFunctionType.php]: add an empty line

File: WorkflowFunctionAdminBundle/Form/Type/WorkflowFunctionType.php [hunk 2]
```diff
@@ -52,6 +56,10 @@ public function configureOptions(OptionsResolver $resolver)
         ));
     }
 
+    protected function getChoices()
```
Comment (alavieille) [inline WorkflowFunctionAdminBundle/Form/Type/WorkflowFunctionType.php:62]: method comment

File: WorkflowFunctionAdminBundle/Form/Type/WorkflowFunctionType.php (hunk 1 above)
Comment (alavieille) [inline WorkflowFunctionAdminBundle/Form/Type/WorkflowFunctionType.php:67]: add an empty line

File: WorkflowFunctionAdminBundle/Tests/Form/WorkflowFunctionTypeTest.php [hunk 3]
```diff
@@ -12,13 +13,23 @@ class WorkflowFunctionTypeTest extends \PHPUnit_Framework_TestCase
 {
     protected $workflowFunctionClass = 'fakeClass';
     protected $workflowFunctionType;
-
+    protected $roleRepositoryInterface;
+    protected $roles;
     /**
```
Comment (alavieille) [inline WorkflowFunctionAdminBundle/Tests/Form/WorkflowFunctionTypeTest.php:19]: add an empty line

**Comment by veolys-nanne** [comment]
done @itkg-alavieille

**Comment by alavieille** [comment]
:+1: when test run

**Pull request merged**